tox.ini

```diff
@@ -37,7 +37,7 @@ external_wheels =
     py38-ci: dist/*cp38*.whl
 setenv =
     COVERAGE_FILE = {env:COVERAGE_FILE:{toxworkdir}/.coverage.{envname}}
-    ci: SNOWFLAKE_PYTEST_OPTS = -vvv --color=yes
+    ci: SNOWFLAKE_PYTEST_OPTS = -vvv
```

Comment:
oh, you are giving up the colored output :)

Reply:
I moved it to `build_test.yaml` so that it would not try to color output on Jenkins anymore :)
However, with this PR `tox` will be swallowing the test outputs unless they fail, because it is going to run environments in parallel.